drivers/png/png_driver_common.cpp

```diff
@@ -125,6 +125,7 @@ Error png_to_image(const uint8_t *p_source, size_t p_size, bool p_force_linear,
 }
 
 Error image_to_png(const Ref<Image> &p_image, Vector<uint8_t> &p_buffer) {
+	ERR_FAIL_COND_V(p_image == NULL, FAILED);
```

Comment:
This should be removed and added to a separate pull request if it fixes a bug.

Reply:
Yes i'm gonna move that to another PR. My editor crashed everytime i import an audio file, this fixed it but i'm not sure what the real bug is.